### ISSUE TITLE
[Accessibility] : Insufficient Border Contrast Ratio for Search Field in Value Help Dialog  (WCAG 2.1 Level AA)

### ISSUE THREAD

**Issue by I553351** [SAP/fundamental-ngx#14183]
### Problem Description:
In the **Fundamental NGX Library (Angular),** the borders associated with ‘Search’ and ‘Synchronize’ button associated with ‘Search’  field within **Value Help Dialog** does not satisfy minimum **contrast ratio of 3:1 , failing to meet  WCAG 2.1 Level AA accessibility checkpoint** 

This issue has been observed in:
✅ Default Theme
✅ Evening Horizon Theme

See Screenshot attached here:  

<img width="1322" height="713" alt="Image" src="https://github.com/user-attachments/assets/5f511954-d307-4a7f-9d6e-9137ab488143" />

****
### ✅ Expected Behavior
All visual boundaries used to identify a UI component and its interactive state should be rendered with a contrast ratio of at least 3:1 against adjacent colors, in accordance with WCAG 2.1 SC 1.4.11 (Non-text Contrast, Level AA).

### ❌ Actual Behavior
The border contrast ratio of the Search input field and related buttons does not meet the 3:1 minimum threshold in the Default and Evening Horizon themes.

****

<img width="499" height="346" alt="Image" src="https://github.com/user-attachments/assets/7c9c5447-aa65-47f9-aba2-59eb981854ee" />

****
**📝 Note / Additional Context:**

_This issue was originally identified and reported by the **Accessibility Review and Compliance Check Team** during an internal accessibility audit of the our application (internal to SAP).

The root cause has been traced to the Fundamental NGX Library (Angular) — specifically the default border styling under the Default and Evening Horizon themes. Since this is a library-level issue and not application-specific, we are raising it here so it can be addressed at the source and benefit all consumers of this library._

****
### Which versions of Angular and Fundamental Library for Angular are affected?

Fundamental NGX Library (current version)

### Affected Component

Value Help Dialog -Search Field 

### Steps to Reproduce (for bugs)

1. Open the **Value help Dialog** using the **Default or Evening Horizon theme.**
 2. Check **Search** input field with an associated action button (e.g., "Search" or "Synchronize").
Inspect the border color of the search input field and button using a contrast analysis tool (e.g., Chrome DevTools, axe, or Colour Contrast Analyser).
Observe that the border contrast ratio falls below 3:1 against the background.

### Priority

Medium

### Declaration

- [x] I'm not disclosing any internal or sensitive information.

**Comment by InnaAtanasova** [comment]
Hi @I553351! Thank you for reporting the issue. This is not something we can fix on our side. We implement the components following the design specs, where every color, spacing, border, etc. is specified. We do not have control over the design and in this case over the contrast issue. You should address this problem to the design team.
